act/discovery/get_armfiles.py

```diff
@@ -31,10 +31,14 @@ def download_data(username, token, datastream, startdate, enddate, time=None, ou
         The name of the datastream to acquire.
     startdate : str
         The start date of the data to acquire. Formats accepted are
-        YYYY-MM-DD, DD.MM.YYYY, DD/MM/YYYY, YYYYMMDD or YYYY/MM/DD.
+        YYYY-MM-DD, DD.MM.YYYY, DD/MM/YYYY, YYYYMMDD, YYYY/MM/DD or
+        any of the previous formats with THH:MM:SS added onto the end
+        (ex. 2020-09-15T12:00:00).
     enddate : str
         The end date of the data to acquire. Formats accepted are
-        YYYY-MM-DD, DD.MM.YYYY, DD/MM/YYYY, YYYYMMDD or YYYY/MM/DD.
+        YYYY-MM-DD, DD.MM.YYYY, DD/MM/YYYY, YYYYMMDD or YYYY/MM/DD, or
+        any of the previous formats with THH:MM:SS added onto the end
+        (ex. 2020-09-15T13:00:00).
     time: str or None
```

Comment:
Do we still want to keep this for legacy purposes or could we remove this and just use the start/enddates above to download one file?

Reply:
I think we can leave it in for legacy purposes... otherwise people's workflows may break